core/storage/base_model/gae_models.py

```diff
@@ -429,6 +430,16 @@ def _compute_snapshot(self):
         return self.to_dict(exclude=['created_on', 'last_updated'])
 
     def _reconstitute(self, snapshot_dict):
+        """Populates the model instance with the snapshot.
+
+        Args:
+            snapshot_dict: dict(str, *). The snapshot with the model
+                property values.
+
+        Returns:
+            self: the class instance. The instance of the model class
```

Comment:
@YashJipkate, here type info is wrong. Instead it should be `VersionedModel ` since it's the instance of the class. Also, remove `self` as we don't write var in returns.